### DISCUSSION TITLE
additionalCartCheckoutInnerBlockTypes: How to remove an inner block

### DISCUSSION THREAD

**Discussion by larsenlarsson** [woocommerce/woocommerce#45001] [WooCommerce, Blocks, and Site Editing]
Hi there 👋 ,

I hope you can help me again.

We want to move the inner block "woocommerce/checkout-actions-block" from "woocommerce/checkout-fields-block" to "woocommerce/checkout-totals-block", because we need the "Place Order" button to be displayed right after the order summary.

![example](https://github.com/woocommerce/woocommerce-blocks/assets/2648926/59d9d3d8-03c6-4202-ad70-e0a390f6a31f)

According [this documentation](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/checkout-block/available-filters.md#allowing-blocks-in-specific-areas-in-the-cart-and-checkout-blocks), we can add "woocommerce/checkout-actions-block" to "woocommerce/checkout-totals-block" by doing this:

```
registerCheckoutFilters( 'myCheckoutFilters', {

	additionalCartCheckoutInnerBlockTypes: ( value, extensions, { block } ) => {

		if ( block === 'woocommerce/checkout-totals-block' ) {
			value.push( 'woocommerce/checkout-actions-block' );
		}
		return value;
	},
} );
```

But how can we remove the inner block correctly from "woocommerce/checkout-fields-block"?
Thank you for your time 😃

**Comment by opr** [comment]
Hi, thanks for posting this question, and showing us your interesting way of using the filter too!

Currently there's no way to remove an inner block like this so I think your best bet would be to hide it with CSS or remove it from the DOM using JS.

Hope this helps!

**Reply by larsenlarsson** [reply]
Hi @opr, thanks for your answer. So, CSS will be the solution then 🥂
